packages/core/src/generatePolicy.js

```diff
@@ -251,7 +255,14 @@ function createModuleInspector(opts) {
    * @param {boolean} includeDebugInfo
    */
   function inspectForGlobals(ast, moduleRecord, packageName, includeDebugInfo) {
-    const commonJsRefs = ['require', 'module', 'exports', 'arguments']
+    const commonJsRefs = [
+      'require',
+      'module',
+      'exports',
+      'arguments',
+      'import',
+      'export',
```

Comment:
this seems incorrect. I can definitely create a function named `import` or use a global named `import` in a cjs module.

Reply:
since this wants a `LavamoatModuleRecord`, it may make sense to stuff the information in there, wouldn't it?

now I am wondering why `ast` is its own param and not `moduleRecord.ast`

Reply:
@naugtur I'm going to sit on this until you respond; I don't want to implement this in e.g., `lavamoat-node` or elsewhere if it's not the right direction.

Reply:
Let's discuss this synchronously.
If we don't know what's esm or cjs in here, how do we inspectESMImports?
Endo doesn't support dynamic imports anyway, so maybe that changes our assumptions?
Does the AST specify the difference between dynamic import and a function named import?